<?xml version="1.0"?>
<launch>

  <!--***********************************************************-->
  <!-- Hi-Ros Skeleton Tracker Node: Parameters override and Run -->
  <!--***********************************************************-->
  <include file="$(find hiros_skeleton_tracker)/launch/default.launch">
    <arg name="node_required"                        value="true"               />
    <arg name="node_name"                            value="node_01"            />

    <arg name="in_skeleton_group_topics"             value="[/topic1, /topic2]" />
    <arg name="out_msg_topic_name"                   value="skeleton_group"     />

    <arg name="camera_frequency"                     value="30"                 />

    <arg name="fixed_delay"                          value="0.1"                />
    <arg name="min_joints"                           value="7"                  />
    <arg name="min_markers_distance"                 value="0.5"                />
    <arg name="max_markers_distance"                 value="100"                />
    <arg name="min_orientations_distance"            value="0.5"                />
    <arg name="max_orientations_distance"            value="100"                />
    <arg name="max_delta_t"                          value="0.05"               />
    <arg name="use_markers"                          value="true"               />
    <arg name="use_marker_velocities"                value="true"               />
    <arg name="use_orientations"                     value="true"               />
    <arg name="use_orientation_velocities"           value="true"               />
    <arg name="velocity_weight"                      value="0.05"               />
    <arg name="weight_distances_by_confidences"      value="false"              />
    <arg name="weight_distances_by_velocities"       value="false"              />

    <arg name="filter_trajectories"                  value="true"               />
    <arg name="filter_cutoff_frequency"              value="6.0"                />
  </include>

</launch>
